core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/FlinkRunner.scala

```diff
@@ -60,31 +60,33 @@ object FlinkRunner extends Runner[CR] {
     // https://github.com/lyft/flinkk8soperator/blob/master/pkg/apis/app/v1beta1/types.go#L110
     // Hence setting a dummy variable @todo: remove this once fixed
     val envConfig = EnvConfig(
-      List(
-        EnvVar(JvmArgsEnvVar, "-DDUMMY=dummy")
+      Some(
```

Comment:
So I guess this can be removed right?

Reply:
Yes! This is what this PR is all about...

Reply:
Cool that's why I mentioned it ;-) Did you test it yet in a cluster?

Reply:
Not yet. I plan to do it by end of the day. I'll leave a comment on my test results